src/components/BookItem.js

```diff
@@ -11,14 +12,45 @@ const BookItem = ({ book }) => {
   return (
     <>
 
-      <li>
-        <h2>{book.title}</h2>
-        <p>{book.author}</p>
-        <h3>{book.category}</h3>
-        <button type="button" onClick={handleRemove}>Remove</button>
+      <li className={styles.listbooks}>
+        <div className={styles.listSection}>
+          <div className={styles.firstcol}>
+            <p className={styles.category}>{book.category}</p>
+            <h2 className={styles.title}>{book.title}</h2>
+            <p className={styles.author}>{book.author}</p>
+            <div className={styles.btncontainer}>
+              <button className={styles.btncol} type="button">
+                Comments
+                <span className={styles.vLine}>|</span>
+              </button>
+              <button className={styles.btncol} type="button" onClick={handleRemove}>
+                Remove
+                <span className={styles.vLine}>|</span>
+              </button>
+              <button className={styles.btncol} type="button">Edit</button>
```

Comment:
- [ ] I can see the `text color` of this section is not matching with the given design guideline. Kindly implement the given color for the specific section. Such as for the texts the color should be `#4386bf`.

Reply:
This change has been done well